src/site/content/en/blog/optimize-fid/index.md

```diff
@@ -0,0 +1,300 @@
+    [React](https://reactjs.org/docs/code-splitting.html#reactlazy),	
+    [Angular](https://angular.io/guide/lazy-loading-ngmodules), and	
+    [Vue](https://vuejs.org/v2/guide/components-dynamic-async.html#Async-Components) provide	
+    abstractions to make it easier to lazy-load at the component-level.
+
+{% Aside %}
+Take a look at [Reduce JavaScript payloads with code
+splitting](/reduce-javascript-payloads-with-code-splitting/) to learn more about code-splitting.
+{% endAside %}
+
+Aside from code-splitting, always use [async or	
+defer](https://javascript.info/script-async-defer) for scripts that are not necessary for	
+critical-path or above-the-fold content.	
+
+```html	
+<script defer src="..."></script>	
+<script async src="..."></script>	
+```	
+
+Unless there is a specific reason not to, all third-party scripts should be loaded with either `defer`	
+or `async` by default.
+
+### Minimize unused polyfills	
+
+If you author your code using modern JavaScript syntax and reference modern browsers APIs, you will	
+need to transpile it and include polyfills in order for it to work in older browsers.	
+
+One of the main performance concerns of including polyfills and transpiled code in your site is that	
+newer browsers shouldn't have to download it if they do not need it. To cut down on the JavaScript	
+size of your application, minimize unused polyfills as much as possible and restrict their usage to	
+environments where they're needed.	
+
+To optimize polyfill usage on your site:	
+
++   If you use [Babel](https://babeljs.io/docs/en/index.html) as a transpiler, use	
+    [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env) to only include the polyfills	
+    needed for the browsers you plan on targeting. For Babel 7.9, enable the	
+    [`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) option to further cut down	
+    on any unneeded polyfills	
++   Use the module/nomodule pattern to deliver two separate bundles (`@babel/preset-env` also	
+    supports this via [`target.esmodules`](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules))	
+
+    ```html	
+    <script type="module" src="modern.js"></script>	
+    <script nomodule src="legacy.js" defer></script>	
+    ```	
+
+    Many newer ECMAScript features compiled with Babel are already supported in environments	
+    that support JavaScript modules. So by doing this, you simplify the process of making sure that	
+    only transpiled code is used for browsers that actually need it.
+
+{% Aside %}
+The [Serve modern code to modern browsers for faster page
+loads](/serve-modern-code-to-modern-browsers/) guide goes into more detail about this topic.
+{% endAside %}
+
+## Break up Long Tasks
+
+If you've already attempted to reduce the amount of JavaScript that loads on a single page, it can
+be useful to break down long-running code  into **smaller, asynchronous tasks**. 
+
+**Long Tasks** are JavaScript execution periods where users may
```

Comment:
I'd reverse these first two sentences. You start by saying why a long task is bad, and then you say what it is. How about:

"Long Tasks are synchronous, uninterruptible chunks of work that take longer than 50 ms for the browser to perform. Any time the main thread is blocked for more than 50 ms, users interacting with the page my find it unresponsive."